zeroex/order_validator.go

```diff
@@ -37,12 +37,13 @@ type getOrdersAndTradersInfoParams struct {
 	Signatures     [][]byte
 }
 
-// OrderInfo represents the order information returned from OrderValidator methods.
+// OrderInfo represents the order information emitted from Mesh
 type OrderInfo struct {
 	OrderHash                common.Hash
 	SignedOrder              *SignedOrder
 	FillableTakerAssetAmount *big.Int
 	OrderStatus              OrderStatus
+	TxHash                   common.Hash
```

Comment:
Can we document what `TxHash` means here? It's not completely obvious, but I think it is something like "the hash of the transaction that caused the order status to change"?